tests/integration/sentinel/transitions/sentinel-transition-error-log.spec.js

```diff
@@ -0,0 +1,27 @@
+const utils = require('@utils');
+const sentinelUtils = require('@utils/sentinel');
+const { setTimeout } = require('timers/promises');
+const { expect } = require('chai');
+
+describe('Sentinel transition error log', async function() {
+  this.timeout(10 * 60 * 1000);
+  after(async () => await utils.revertSettings(true));
+
+  it('log error for unkown transition', async () => {
+    const unknownTransitionPattern = /Unknown transition "something"/;
+    const settings = {
+      transitions: {
+        something: true
+      }
+    };
+
+    const collectLogs = await utils.collectSentinelLogs(unknownTransitionPattern);
+    await utils.updateSettings(settings, 'sentinel');
+    await sentinelUtils.getCurrentSeq();
```

Comment:
I'm uncertain what the purpose of this line of code is.

Reply:
not needed. Removed this.